GUI/DeusEx/Classes/MenuChoice_ItemRefusal.uc

```diff
@@ -0,0 +1,18 @@
+class MenuChoice_ItemRefusal extends DXRMenuUIChoiceBool;
+
+var class<Inventory> refusedItems[6];
+
```

Comment:
we could also do something like

```

function SaveSetting()
{
    Super.SaveSetting();
    SetRefusals();
}
```

if we change the `SetRefusals()` function to do `class'DXRLoadouts'.static.UnsetRefuseItem(itemClass);` when we're false instead of just early returning. I think people might expect changing the setting would immediately apply to the current game.

Reply:
That worked great. Toggling a refusal button on then off again will unset any of those items that were manually set before, but I think that's acceptable and probably not very surprising.